include/quicr/gap_check.h

```diff
@@ -32,18 +32,25 @@ namespace quicr {
         std::ostringstream log_s;
 
         if (last_group_id != 0 && last_object_id != 0) {
```

Comment:
Could early return and then the ret of the function wouldn't have to be tabbed over so much. But that's not important to this PR.